ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -21,8 +21,7 @@ function check_if_hypershift_env () {
     elif [ -f "${SHARED_DIR}/mgmt_kubeconfig" ]; then
         export KUBECONFIG="${SHARED_DIR}/mgmt_kubeconfig"
     else
-        echo "This idp-htpasswd step is being run as a day-2 operation for a HyperShift guest cluster. We need the kubeconfig of management cluster, but it does not exist!"
-        exit 1
+        export KUBECONFIG="${SHARED_DIR}/kubeconfig"
```

Comment:
```suggestion
        echo 'The nested_kubeconfig is detected to be existent, thus this idp-htpasswd step is being considered as a day-2 operation for a HyperShift guest cluster. For this, we need a management kubeconfig (hs-mc.kubeconfig or mgmt_kubeconfig) to perform the IDP configuration within a HostedCluster resource. But neither management kubeconfig is detected! So, falling back to consider it is not a HyperShift guest cluster by marking IS_HYPERSHIFT_ENV="no". And treating it just as normal OpenShift cluster.'
        IS_HYPERSHIFT_ENV="no"
        export KUBECONFIG="${SHARED_DIR}/kubeconfig"
```

Reply:
In this pr, it means in some hypershift jobs to run management cluster testcases, there is only ${SHARED_DIR}/kubeconfig that is mgmt cluster 's kubeconfig.  We should use ${SHARED_DIR}/kubeconfig directly while it is still a hypershift env.

So  we should not add this one: IS_HYPERSHIFT_ENV="no"

Reply:
https://github.com/openshift/release/pull/54651/files#diff-6d907a29b0f0f0fbaa8c73358f570a3a0f616673087017b53af957772f8e9f91R37-R50 defines the usage of IS_HYPERSHIFT_ENV=yes and IS_HYPERSHIFT_ENV=no.

Reply:
https://github.com/openshift/release/pull/54651/files#diff-6d907a29b0f0f0fbaa8c73358f570a3a0f616673087017b53af957772f8e9f91R37-R50 defines the usage of IS_HYPERSHIFT_ENV=yes and IS_HYPERSHIFT_ENV=no :
If IS_HYPERSHIFT_ENV=yes, IDP will be configured in hostedcluster resource's .spec.configuration.oauth.identityProviders..
If IS_HYPERSHIFT_ENV=no, IDP will be configured in `oauth/cluster` resource's .spec.identityProviders.
Maybe you can rename IS_HYPERSHIFT_ENV to be IS_HYPERSHIFT_GUEST_ENV to be more clear?